crankshaft-engine/src/service/runner/backend/generic/driver.rs

```diff
@@ -143,13 +143,13 @@ async fn run_local_command(command: String, config: &Config) -> Result<Output> {
 
     // NOTE: this is cloned because `default()` is only implemented on the owned
     // [`Locale`] type (not a reference).
-    let command = match config.shell().cloned().unwrap_or_default() {
-        Shell::Bash => Command::new("/usr/bin/env")
+    let command = match config.shell() {
+        Some(Shell::Bash) | None => Command::new("/usr/bin/env")
```

Comment:
Nit: I wonder if it makes sense to still call the `Default` implementation here, as, if we change it in the future, this adds another place it would need to be updated.

Reply:
Fixed.